src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -646,6 +670,7 @@ private String startBulkCaseBookPDFJob(String schema,
 			String accessToken = (String) request.getSession().getAttribute("accessToken");
 			servletContext.setAttribute("accessToken", accessToken);
 			servletContext.setAttribute("studyID", study.getStudyId()+"");
+			servletContext.setAttribute("pdfHeader", pdfHeader);
```

Comment:
Why set this as a servlet context attribute and not pass it in as a method argument?

Reply:
Also, can pdf header be calculated within the Future?

Reply:
changed --- after discuss, we decided to pass in Study object as method parameter